simtel_baseline.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+
+def fill_baseline_r0(event_stream, n_bins0=5, n_bins1=10, method='data'):
```

Comment:
Ah .. I just see that this function is basically modelled after `digicampipe.calib.camera.random_triggers.fill_baseline_r0()`, right?

I assume you had to write your own function here, since the other `fill_baseline_r0` did somehow not work for you. I think this is fine, but we should merge these two functions as soon as possible ... otherwise we end up with 3 functions for every task, one for zfits one for hdf5 one for hession ... and each of these functions might work **nearly** identical .. but not quite .. that quickly turns into chaos.... I fear.

We assume we can do this later .. but we should keep it in mind. If we merge this as it is, I'd be happy if we could open an issue, kind of as a reminder to merge this stuff.

Reply:
Agree.. for now I I have just renamed and split the two methods into two functions